src/fmeval/transforms/common.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, input_keys: List[str], output_keys: List[str], prompt_templat
 
     @validate_call
     def __call__(self, record: Dict[str, Any]) -> Dict[str, Any]:
-        """Augments the input record with LLM prompts and returns said record.
+        """Augment the input record with LLM prompts and returns said record.
```

Comment:
nit: umm isn't the old grammar correct? Subject in the sentence is input record i.e. singular

Reply:
Oh, I'm just switching to using the imperative form of the verb. I think either way is fine; I'll just have make sure we're consistent across the docstrings.